ia-configs/IA.zebra.json

```diff
@@ -1,33 +1,42 @@
 {
-    "_note": "The hierarchy of this file is genus->species->ia_class->_id_conf (id config). detect_config is under Genus->species. Keys with a leading underscore and their children are non-biological categories of config, like _id_conf and _detection_conf. Keys without leading underscores are semantic lookups that are derived from Wildbook data, e.g. Megaptera.novaeangliae.whale_fluke.",
-    "_default": "@Equus.quagga",
-    "Equus": {
-      "grevyi": "@Equus.quagga",
-      "quagga+quagga": "@Equus.quagga",
-      "quagga": {
-        "_default":{
-          "_id_conf": "@Equus.quagga.zebra._id_conf"
-        },
-        "_detect_conf": [
-          {
-            "labeler_algo": "densenet",
-            "labeler_model_tag": "zebra_v1",
-            "model_tag": "ggr2",
-            "nms_thresh": 0.4,
-            "sensitivity": 0.4
-          }
-        ],
-        "zebra_grevys": "@Equus.quagga.zebra",
-        "zebra":{
-          "_id_conf": [
-            {
-              "query_config_dict": {
-                "sv_on":false
-              },
-              "description": "HotSpotter pattern-matcher"
-            }
-          ]
-        }
+  "_README": "A value starting with '@' indicates an absolute path from the top of this config file, and parsing that @-link results in the value of that absolute path. The first half of this file (top-level _detectors and _identifiers) defines the various algorithms we use, including the configs we send to SAGE and some metadata about each config. Each algo config is under a name-key, e.g. the african_terrestrial detector.  \n The second half of the file defines the pipeline for each species by referencing the previously-defined algorithms, in the hierachy genus->species->detectors, and genus->species->iaClass->identifiers. The values of these species configurations should be @-links but literals are also supported by the parser.",
+  "_detectors": {
+    "african_terrestrial": {
+      "name": "African terrestrial mammal detector",
+      "description": "Trained on zebras, giraffes, lions, hyenas, leopards, cheetahs, and wild dogs.",
+      "config_dict": {
+        "start_detect": "https://kaiju.dyn.wildme.io:5010/api/engine/detect/cnn/lightnet/",
```

Comment:
NotAnExpertOnThis but surely the hostname shouldn't be specified in the config file.

Reply:
well, this is the way _old world_ does things.   and it raises an interesting question:  how will we override the "default" behavior of _"use your own [docker] acm/wbia"_ ?   it seems to me that one of the goals of these .json conf files will be that they can be used *universally* on any codex.  so maybe we need to have the urls **be relative** and set the **wbia host** elsewhere in the site setup (with default being something like `http://acm` -- or whatever docker sees?

prob a good slack chat, if we dont want to derail this PR too much.

Reply:
I tried to pull a bunch of this out into the global file in the same directory

Reply:
I removed the hostname so it's just the endpoint now.